src/main/js/electron/templates/welcome.html

```diff
@@ -86,4 +108,4 @@
 </div>
 
 
-{% endblock %}  
+{% endblock %} 
```

Comment:
No newline at end of file -- better to have a newline at the end. A style  thing - sometimes it's inconvenient when there's no newline, for example with printing several files behind each other: The first line of the next file would be on the same line as the last line of the first, because the last line doesn't have a newline character at the end.
Please add it :-)